src/schemas/block.json

```
@@ -125,7 +125,8 @@
 	"BlockTag": {
 		"title": "Block tag",
 		"type": "string",
-		"enum": ["earliest", "latest", "pending"]
+		"enum": ["earliest", "finalized", "safe", "unsafe", "pending"],
+		"description": "`earliest`: The lowest numbered block the client has available.\n`finalized`: The most recent block that cannot be reorged outside of a hard fork.\n`safe`: The most recent block that is will not reorg unless there is an active ongoing attack.\n`unsafe`: The most recent block the client has seen that it believes will be part of the canonical chain (high probability of being reorged out).\n`pending`: A sample next block built by the client containing what it believes will be the set of transactions likely to show up in that block.\n`latest`: **DEPRECATED** Will be removed at some point in the future, currently an alias for `unsafe` or `safe`."
```

Removes newlines and uses `;` instead (newlines apparently aren't supported in OpenRPC).

```suggestion
		"description": "`earliest`: The lowest numbered block the client has available; `finalized`: The most recent block that cannot be reorged outside of a hard fork; `safe`: The most recent block that is will not reorg unless there is an active ongoing attack; `unsafe`: The most recent block the client has seen that it believes will be part of the canonical chain (high probability of being reorged out); `pending`: A sample next block built by the client containing what it believes will be the set of transactions likely to show up in that block; `latest`: **DEPRECATED** Will be removed at some point in the future, currently an alias for `unsafe` or `safe`."
```